browser_history/generic.py

```diff
@@ -175,26 +191,63 @@ def fetch(self, history_paths=None, sort=True, desc=False):
                 date_histories = [
                     (
                         datetime.datetime.strptime(d, "%Y-%m-%d %H:%M:%S").replace(
-                            tzinfo=_local_tz
+                            tzinfo=self._local_tz
```

Comment:
Any particular reason for adding this into the class instead of keeping it a global?

Reply:
Stupid reason but since i used a bookmark parser function inside the class, pylint gave me warnings if I didnt use the self variable, thats why I did it xD